### PR TITLE
Add button to open user feedback form

### PR DESCRIPTION
**Pull request by danfeltham**
## Summary
This pull request is to enable users to provide feedback on the application from the application itself. It adds a button to the project page that opens a Google Form that asks a few questions about the app. [Here ](https://docs.google.com/forms/d/1ohv829YWfNuL9wcHhgTmjsMJKdJT5Qkwg6iJU9JdQHs/edit?usp=sharing)is a link to the feedback form.

<hr/>

### Why is this change needed?
It is done to allow for effective user testing, particularly if the application is shared outside of the known network.

<hr/>

### What did you change?
Added a button to the bottom right of the project page.

<hr/>

### Does it depend on any other changes/PRs?
Nope.

<hr/>

### Screenshots of UI changes, if any
image

<hr/>

### Links to any design documents which you made/used in order to make the decisions that you made.
I placed the button there based on UI designs sent to the Phoenix channel but decided it didn't belong in the video controller bar at the top.
<hr/>

### OS

- [ ] Linux
- [ ] MacOS
- [x] Windows

### CODE REVIEW

File: src/renderer/pages/Project.tsx [hunk 1]
```diff
@@ -29,6 +31,9 @@ const ProjectPage = () => {
   const { isExporting, exportProgress } = useSelector(
     (store: ApplicationStore) => store.exportIo
   );
+  const openUserFeedback = () => {
+    ipc.openExternalLink('https://forms.gle/YNozsPZxF4kxSuud8');
```
Comment (chloebrett) [inline src/renderer/pages/Project.tsx]: I reckon we should put all URL constants like this in one place - make a constants file and put this as well as the other URLs (e.g. the link to AssemblyAI signup) in there

Comment (jhea0008) [reply src/renderer/pages/Project.tsx]: Yep, great idea. Do you want to create a file in this PR?

Comment (danfeltham) [reply src/renderer/pages/Project.tsx]: Done

**Review by jhea0008 — APPROVED** [review]

**Review by kateahyde — DISMISSED** [review]
I don't think this matches the designs @emmapainter and @rileykeane created?

**Comment by danfeltham** [comment]
> I don't think this matches the designs @emmapainter and @rileykeane created?

Updated the designs to: 
image
 per the design Emma showed.

**Review by chloebrett — APPROVED** [review]
Code looks good

**Comment by rileykeane** [comment]
@emmapainter do we want the feedback button on the home page too?

**Comment by rileykeane** [comment]
@Aretle a few minor changes to the design of the button. 

- Make the button slightly smaller at 25px x 25px
- Add the word '_Feedback_' underneath  

Frame 124

**Comment by danfeltham** [comment]
> @Aretle a few minor changes to the design of the button.
> 
> * Make the button slightly smaller at 25px x 25px
> * Add the word '_Feedback_' underneath
> 
> Frame 124

Done

**Review by rileykeane — CHANGES_REQUESTED** [review]
@Aretle can you please bump the font size for the word '_feedback_' up to 12px.

**Comment by danfeltham** [comment]
> @Aretle can you please bump the font size for the word '_feedback_' up to 12px.

Done

image

**Review by rileykeane — APPROVED** [review]
bellissimo 🎉

**Pull request merged**